docs/examples/server.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Seeing `BEGIN PRIVATE KEY` anywhere in a PR is somewhat worrisome, as it provides an opportunity for naive users to re-use publicly known keys.   Not to mention the fact that github security scanners will forever be notifying the org admins about this "security problem" :).  I'd really rather that this go into a key-bootstrapping script.